rmc/pipeline/regional_constraint.py

```diff
@@ -344,128 +353,178 @@ def main(args):
+                            simul_break_ht.breakpoints[1] + 1,
+                            simul_break_ht.section.split("_")[2],
+                        ),
+                        hl.if_else(
+                            simul_break_ht.locus.position
+                            > simul_break_ht.breakpoints[0],
+                            hl.format(
+                                "%s_%s_%s",
+                                simul_break_ht.section.split("_")[0],
+                                simul_break_ht.breakpoints[0] + 1,
+                                simul_break_ht.breakpoints[1],
+                            ),
+                            hl.format(
+                                "%s_%s_%s",
+                                simul_break_ht.section.split("_")[0],
+                                simul_break_ht.section.split("_")[1],
+                                simul_break_ht.breakpoints[0],
+                            ),
+                        ),
+                    )
+                )
+                simul_break_ht = simul_break_ht.key_by(
+                    "locus", section=simul_break_ht.section_1
+                ).drop("section_1", "breakpoints")
+            else:
+                simul_exists = False
+                logger.info(
+                    "No sections in round %i had breakpoints in simultaneous breaks search.",
+                    args.search_num,
                 )
-            )
-            # Filter to sections with simultaneous breaks
-            single_no_break_ht = single_no_break_ht.annotate(
-                breakpoints=simul_by_section_ht[single_no_break_ht.section].breakpoints
-            )
-            simul_break_ht = single_no_break_ht.filter(
-                hl.is_defined(single_no_break_ht.breakpoints)
-            )
 
-            logger.info("Annotating new sections and re-keying for next search...")
-            single_break_ht = hl.read_table(
+            if file_exists(
                 single_search_round_ht_path(
                     is_rescue=args.is_rescue,
                     search_num=args.search_num,
                     is_break_found=True,
                     is_breakpoint_only=False,
                 )
-            )
-            single_break_ht = single_break_ht.annotate(
-                section_1=hl.if_else(
-                    single_break_ht.locus.position > single_break_ht.breakpoint,
-                    hl.format(
-                        "%s_%s_%s",
-                        single_break_ht.section.split("_")[0],
-                        single_break_ht.breakpoint + 1,
-                        single_break_ht.section.split("_")[2],
-                    ),
-                    hl.format(
-                        "%s_%s_%s",
-                        single_break_ht.section.split("_")[0],
-                        single_break_ht.section.split("_")[1],
-                        single_break_ht.breakpoint,
-                    ),
+            ):
```

Comment:
```suggestion
            single_break_path = single_search_round_ht_path(
                is_rescue=args.is_rescue,
                search_num=args.search_num,
                is_break_found=True,
                is_breakpoint_only=False,
            )
            if file_exists(single_break_path):
```

Reply:
will add the `single_break_path` code change, but don't we still need the section update?

Reply:
(I also don't see this `single_break_ht = hl.read_table(` on line 382, so not sure what happened there)

Reply:
I didn't make the change to the section update code, that was already there. I just added the `single_break_path` stuff

Reply:
Things just got moved around, everything is still in there

Reply:
ah gotcha. I'm going to guess this was a weird formatting thing